### PR TITLE
Add RSpec FakeSchemaRegistryServer test helper

### PR DESCRIPTION
**Pull request by jturkel**
@tjwp - you're prime

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -251,6 +251,13 @@ The following types/features are not supported for generated models:
   another type, is supported.
 - Reused models for nested records: Currently an anonymous model class is
   generated for each subrecord.
+  
+### Test Utilities
+
+Tests that leverage the Messaging API for encoding/decoding will need to talk to schema registry.
```
Comment (tjwp) [inline README.md]: "a schema registry"? "the schema registry"?

File: lib/avromatic/rspec.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require 'webmock/rspec'
+require 'avro_turf/test/fake_schema_registry_server'
+
+RSpec.configure do |config|
+  config.before(:each) do
+    WebMock.stub_request(:any, /^#{Avromatic.registry_url}/).to_rack(FakeSchemaRegistryServer)
+    FakeSchemaRegistryServer.clear
+  end
+end
```
Comment (tjwp) [inline lib/avromatic/rspec.rb]: There aren't currently any other test utilities but I'm inclined to make this `lib/avromatic/rspec/fake_registry_server.rb`

File: README.md (hunk 1 above)
Comment (jturkel) [inline README.md]: After offline discussion we're going to remove the README change and leave this undocumented for now.

File: lib/avromatic/rspec.rb (hunk 2 above)
Comment (jturkel) [inline lib/avromatic/rspec.rb]: We talked it through and we're going to leave this as-is for now and split it out later if we add more test utilities.

**Pull request closed without merging**